### PR TITLE
sexor

### PR DESCRIPTION
**Pull request by S3x0r**
Bello!

### CODE REVIEW

File: data/delegates.json [hunk 1]
```diff
@@ -343,5 +343,22 @@
     "poolMember": false,
     "required": false,
     "featured": false
+  },
+  { 
+    "delegateName": "sexor",
```
Comment (4fryn) [inline data/delegates.json]: `minions` ?

**Comment by 4fryn** [comment]
please also add yourself to `data/groups.json`

**Comment by S3x0r** [comment]
You mean to builders group?

**Comment by S3x0r** [comment]
Done

**Comment by 4fryn** [comment]
git push? ;)

**Comment by 4fryn** [comment]
Ah, I see, #66

**Pull request merged**